src/backend/core/api/serializers.py

```diff
@@ -131,15 +133,20 @@ class TeamSerializer(serializers.ModelSerializer):
 
     abilities = serializers.SerializerMethodField(read_only=True)
     accesses = TeamAccessSerializer(many=True, read_only=True)
+    slug = serializers.SerializerMethodField()
```

Comment:
actually this is not required. You should use the `slug` field from the model directly...

Reply:
Well, it is indeed useless if you only use the model and its modified `save`. In cases where you use the serializer directly though, the slug field comes empty without a `get_slug` method. And I need to define "slug" here to do that.